packages/sites/ortho-site/webapp/wdkCustomization/js/client/records/Sequences.tsx

```diff
@@ -71,7 +71,15 @@ export function RecordTable_Sequences(
     }))
     // and remove a raw HTML checkbox field - we'll use Mesa's built-in checkboxes for this
     // and an object-laden 'sequence_link' field - the ID seems to be replicated in the full_id field
-    .filter(({ key }) => key !== 'clustalInput' && key !== 'full_id');
+    .filter(
+      ({ key }) =>
+        key !== 'clustalInput' && key !== 'full_id' && key !== 'taxon_abbrev'
+    )
+    // rename some headings (shouldn't this be done on the back end?)
+    .map((column) => ({
+      ...column,
+      name: column.name === 'Taxon' ? 'Clade' : column.name,
+    }));
```

Comment:
All of these changes can be done on the backend, in the WDK model for ortho.

Reply:
Ah yes, thanks. I've done it here
https://github.com/VEuPathDB/OrthoMCLModel/pull/5